templates/manifest.json

```diff
@@ -121,22 +121,24 @@
             ]
         },
         {
-            "id": "js-start",
-            "name": "getting_started_node",
-            "label": "Start with JavaScript",
-            "description": "A simple JavaScript example of core Actor and Apify SDK features. It reads and validates user input with schema, computes a result and saves it to storage.",
+            "id": "js-scrape-single-page",
```

Comment:
Why not keep the `js-start` ID and Start with JavaScript name? That was kinda the point, that the "primary start template" would be named `start` so that everyone understands it's the getting started example. How are the users gonna know what they should start with now?

Reply:
@jbartadev Any news on this? I feel quite strongly about this one. We should have exactly one "getting started" template for each language. It's standard in all documentation and tutorials incl. Vercel (I know you take inspiration from it)
![CleanShot 2023-05-31 at 11 08 24@2x](https://github.com/apify/actor-templates/assets/23726914/f30d3f84-bfba-450c-9d67-cfb25ccd17f4)

Reply:
Yeah, sorry I forget to reply here. I agree with you for sure. We just stopped with the work on the template until we have figured out the readme structure and logic. I'll update it.